includes/class-wc-stripe-webhook-handler.php

```
@@ -553,8 +553,8 @@ public function process_webhook_capture( $notification ) {
 				if ( $this->is_partial_capture( $notification ) ) {
 					$partial_amount = $this->get_partial_amount_to_charge( $notification );
 					$order->set_total( $partial_amount );
-					$refund_object = $this->get_refund_object( $notification );
-					$this->update_fees( $order, $refund_object->balance_transaction );
+					$refund = new WC_Stripe_Refund( $this->get_refund_object( $notification ) );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '556', column '20'). (undefined)

(MissingImport)